tests/Functional/Query/QueryBuilderTest.php

```diff
@@ -380,4 +461,15 @@ private function platformSupportsSkipLocked(): bool
 
         return ! $platform instanceof SQLitePlatform;
     }
+
+    private function platformSupportsCTEs(): bool
+    {
+        $platform = $this->connection->getDatabasePlatform();
+
+        if (! $platform instanceof MySQLPlatform) {
+            return true;
+        }
+
+        return $this->connection->getServerVersion() >= '8.0.11';
```

Comment:
MySQL 8.0.0 - 8.0.10 were prereleases. I don't think we need to implement any special handling for them. We're not testing against those releases anyway.

```suggestion
        return ! $platform instanceof MySQLPlatform || $platform instanceof MySQL80Platform;
```